### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Sergey Komarov

### DISCUSSION THREAD

**Discussion by ShadySeko** [heig-vd-dai-course/heig-vd-dai-course#91] [Show and tell]
Hello! During this practical work I will implement a CLI that can reverse a text! 
Ex: "Hello World!" ---> "!dlroW olleH"

UPDATE : this CLI has evolved, it can now also swap all occurrences of a character in a text file by another character.

[Here's the link to my repository for this project](https://github.com/ShadySeko/MyCLI2)

f80f8776583b3cf049f75f17cc1e3389cb173e15 is the latest commit hash

**Comment by ludelafo** [comment]
Great!

As you still have time, I would try to add another command to your CLI for another file processing. But this start is good already!

**Reply by ShadySeko** [reply]
Of course! I've just updated my work with another command :)

**Reply by ludelafo** [reply]
Great! Feel free to add more commands in the future if you want to!
